src/libraries/System.Net.Http/tests/FunctionalTests/ResponseStreamZeroByteReadTests.cs

```diff
@@ -114,29 +114,29 @@ public async Task ZeroByteRead_IssuesZeroByteReadOnUnderlyingStream(StreamConfor
                             cert,
                             clientCertificateRequired: true,
                             enabledSslProtocols: SslProtocols.Tls12,
-                            checkCertificateRevocation: false).WaitAsync(TimeSpan.FromSeconds(10));
+                            checkCertificateRevocation: false).WaitAsync(TimeSpan.FromSeconds(TestHelper.PassingTestTimeoutMilliseconds));
```

Comment:
I believe it should be `FromMilliseconds` now here and below as timeout's name suggests it's in millis 😄
```suggestion
                            checkCertificateRevocation: false).WaitAsync(TimeSpan.FromMilliseconds(TestHelper.PassingTestTimeoutMilliseconds));
```

P.S.: Should we extract it to some extension method in TestHelper? Now it is rather long to be added everywhere 😄

Reply:
We already have TestHelper.PassingTestTimeout... this can just use that instead of `TimeSpan.FromMilliseconds(TestHelper.PassingTestTimeoutMilliseconds)`, which will also fix the cited bug.

Reply:
my bad. I did quick search & replace last nigh without much thinking. I put in `TestHelper.PassingTestTimeout` as I should have.